worldedit-bukkit/src/main/java/com/boydti/fawe/bukkit/adapter/mc1_16_5/TuinityRelighterFactory_1_16_5.java

```diff
@@ -0,0 +1,21 @@
+package com.boydti.fawe.bukkit.adapter.mc1_16_5;
+
+import com.boydti.fawe.beta.IQueueChunk;
+import com.boydti.fawe.beta.IQueueExtent;
+import com.boydti.fawe.beta.implementation.lighting.NullRelighter;
+import com.boydti.fawe.beta.implementation.lighting.Relighter;
+import com.boydti.fawe.beta.implementation.lighting.RelighterFactory;
+import com.boydti.fawe.object.RelightMode;
+import com.sk89q.worldedit.world.World;
+import org.bukkit.Bukkit;
+import org.bukkit.craftbukkit.v1_16_R3.CraftWorld;
+import org.jetbrains.annotations.NotNull;
+
+public class TuinityRelighterFactory_1_16_5 implements RelighterFactory {
+    @Override
+    public @NotNull Relighter createRelighter(RelightMode relightMode, World world, IQueueExtent<IQueueChunk> queue) {
+        org.bukkit.World w = Bukkit.getWorld(world.getName());
+        if (w == null) return NullRelighter.INSTANCE;
+        return new TuinityRelighter_1_16_5(((CraftWorld) w).getHandle(), queue);
+    }
+}
```

Comment:
Given tuinity only targets the latest version (like paper), I should think it's fine for any paper/tuinity specifical "modules" to only target "tuinity"-latest rather than specific versions? Makes our lives easier.

Reply:
Starlight was added in 1.16.5 (afaik). Not sure how relevant that will be for future releases, but it can be abstracted a bit if we want to support multiple version without duplicating everything.

Reply:
I guess it might be fine, though I do still think creating separate tuinity/paper modules would be good (discussion for another time)